examples/mqtt/README.md

```diff
@@ -0,0 +1,75 @@
+# Using MQTT as Transport in LIOTA
+
+LIOTA offers MQTT protocol as transport at both Device & DCC ends via ![MqttDeviceComms](/liota/device_comms/mqtt_device_comms.py) & ![MqttDccComms](/liota/dcc_comms/mqtt_dcc_comms.py).
+
+## Security
+LIOTA's Mqtt client supports both Username-password based authentication & certification based authentication.  Users can configure appropriate parameters using these objects
+while initializing MqttDeviceComms and MqttDccComms:
+
+* ![EdgeSystemIdentity & RemoteSystemIdentity](/liota/lib/identity/identity.py)
```

Comment:
As discussed, this will be part of transports.  So, this will not be added to `README.md`